### ISSUE TITLE
Explorer aria label should contain name of top level folder (or workspace in MR scenario)

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#160788]
**GitHub Tags**:#A11yMAS;#A11yTCS;#A11ySev2;#DesktopApp;#Win32;#Visual Studio Code Client;#BM-VisualStudioCodeClient-Win32-Aug2022;#WCAG1.3.1;#NVDA; #ScreenReader;#JAWS;#AILimited;#Element:Button;

## Environment and OS details:
Application Name:  Visual Studio Live Share
Application version: 1.56.2 (user setup)
Windows Version: Win10 

## Repro Steps:

1. Launch VS Code
2. Open a project (e.g., vsls-guestbook) in VSCode.
3. Tab till “sign in” button.
4. Tab till "Live share" under Extensions and hit enter
5. Turn on the NVDA and tab to drop down 'Visual Studio live share work' screen.

## Actual Result:
The label of the 'Visual Studio live share work….' is being read as 'folder section button' when the screen reader focus moves to it using tab key.

Similar issue is observed with JAWS.

## Expected Result:
The screen reader should read it as 'Visual Studio live share work button' when screen reader focus moves to it.

## User Impact:
People who uses screen reader may face difficulty while activating an object when the accessible label is different than that of the visible labels.

## Attachment:

https://user-images.githubusercontent.com/24580570/189897653-58f80b70-3aa9-4435-9106-21ef4e07a72f.mp4

**Comment by fsteffi** [comment]
logged for primary bug 1335240

**Comment by isidorn** [comment]
I think this issue makes sense.
In short - aria label of the explorer view should contain the name of the top folder. Currently it is just "Folders". So it should be dynamic.
Assigning to @lramos15

**Comment by lramos15** [comment]
@isidorn Actually not too sure how to fix this one. The section being read out is the static section name of the view pane

https://github.com/microsoft/vscode/blob/8636541385743e44f861c46e9dfda74de76e8179/src/vs/workbench/contrib/files/browser/explorerViewlet.ts#L143

https://github.com/microsoft/vscode/blob/8636541385743e44f861c46e9dfda74de76e8179/src/vs/base/browser/ui/splitview/paneview.ts#L130


We do have an aria label within that section header with the name of the folder which dynamically updates but that never gets focused. The pane registration is static and we don't know the folder name at that time


https://github.com/microsoft/vscode/blob/8636541385743e44f861c46e9dfda74de76e8179/src/vs/workbench/contrib/files/browser/views/explorerView.ts#L265

**Comment by isidorn** [comment]
@lramos15 there's the [`updateHeader`](https://github.com/microsoft/vscode/blob/8636541385743e44f861c46e9dfda74de76e8179/src/vs/base/browser/ui/splitview/paneview.ts#L273) method in the `paneview.ts`.  Update header should set the aria label of the header. And the explorer should call updateHeader every time the content of the header changes (which it might do already).

**Comment by fsteffi** [comment]
@lramos15 

could you please the bug in open state the bug can be closed only by trusted tester after verification 
right now the fix is not present in the app

Version: 1.72.0-insider (user setup)
Commit: 1f148906f1ece5c83a60ed8e211c9f387188dca3
Date: 2022-09-23T05:17:37.024Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

![image](https://user-images.githubusercontent.com/24580570/194807356-b301c5b5-dc72-460f-aea7-d6120bc53391.png)

**Comment by isidorn** [comment]
@fsteffi please look at the label that the VS Code bot has added to the issue. It says "unreleased". Once the bot adds the label "released" that means the fix is present.
We can not leave the issue open for the tester to close this since this does not align with the rest of our VS Code processes.
In case the bug is not fixed after it is released, just comment on the issue and we will reopen it. Thanks!
